### PR TITLE
Update and fix french locales

### PR DESCRIPTION
**Pull request by Mathu-lmn**
### Goal of this PR

This PR updates the french translation for the latest interface version, adding MANY missing labels

Nb. If this PR isn't supposed to be submitted here, please redirect me to where I should send it, thanks !

### How is this PR achieving the goal

By translating the locale-en.json in french

### This PR applies to the following area(s)

Translation, interface

### Checklist

- [x] Code compiles and has been tested successfully.
- [x] Code explains itself well and/or is documented.
- [x] My commit message explains what the changes do and what they are for.
- [x] No extra compilation warnings are added by these changes.

### CODE REVIEW

File: ext/cfx-ui/src/assets/languages/locale-fr.json [hunk 1]
```diff
@@ -1,113 +1,303 @@
...
+  "#AuthUI_Header": "Compte",
+  "#AuthUI_CancelAndForget": "Annuler et ne pas demander à nouveau",
+  "#AuthUI_CancelAndForget_Title": "Vous pourrez toujours le lier dans les paramètres",
+  "#ExitToDesktop": "Quitter vers le bureau",
+  "#ServerList_Name": "Nom",
+  "#ServerList_Players": "Joueurs",
+  "#ServerList_Ping": "Ping",
+  "#ServerList_Filter": "Filtrer",
+  "#ServerList_Filter_Reset": "Réinitialiser",
+  "#ServerList_Browse": "Parcourir tous les serveurs",
```
Comment (PsychoShock) [inline ext/cfx-ui/src/assets/languages/locale-fr.json:37]: `Parcourir l'ensemble des serveurs` seems a "better" french again.

File: ext/cfx-ui/src/assets/languages/locale-fr.json [hunk 2]
```diff
@@ -1,113 +1,303 @@
...
+  "#ServerList_Favorites": "Favoris",
+  "#ServerList_History": "Historique",
+  "#ServerList_Premium": "Supporters",
+  "#ServerList_DirectConnect": "Connexion directe",
+  "#ServerList_Tags": "Tags",
+  "#ServerList_StreamerModeWarning": "Le mode Streamer est activé - l'entrée est masquée",
+  "#ServerList_SearchHint": "Rechercher ou entrer une adresse IP",
+  "#ServerList_SearchHintWeb": "Recherche",
+  "#ServerList_SearchHint2": "Rechercher ou taper > et entrer une adresse IP",
```
Comment (PsychoShock) [inline ext/cfx-ui/src/assets/languages/locale-fr.json:46]: Why `>`

File: ext/cfx-ui/src/assets/languages/locale-fr.json [hunk 3]
```diff
@@ -1,113 +1,303 @@
...
+  "#Settings_InProcessGpuDesc": "Correction du 'lag' de l'UI à une utilisation élevée du GPU, mais peut causer des problèmes de fiabilité en cas de crash du GPU (nécessite un redémarrage)",
+  "#Settings_FixedSizeNUI": "Interface utilisateur NUI de taille fixe",
+  "#Settings_FixedSizeNUIDesc": "Forcer l'UI en jeu à une résolution de 1920x1080, mise à l'échelle/letterbox pour s'adapter (pour les serveurs avec une UI supposant 1920x1080)",
+  "#Settings_NoiseSuppression": "Réduction du bruit du chat vocal",
+  "#Settings_NoiseSuppressionDesc": "Activer la réduction du bruit de fond (utilisant RNNoise) pour le chat vocal intégré",
+  "#Settings_ConnectedProfiles": "Identités liées",
+  "#Settings_UpdateChannel": "Canal de mise à jour",
+  "#Settings_UpdateChannelDesc": "Essayez de nouvelles fonctionnalités et modifications avant leur lancement. Il y a un risque de dysfonctionnement.",
+  "#Settings_LowPerfMode": "Réduire le flou (pour les systèmes peu performants)",
+  "#Settings_LowPerfModeDesc": "",
```
Comment (PsychoShock) [inline ext/cfx-ui/src/assets/languages/locale-fr.json:138]: Missing here?

**Review by PsychoShock — COMMENTED** [review]
Seem's good for 98%.

File: ext/cfx-ui/src/assets/languages/locale-fr.json [hunk 4]
```diff
@@ -1,113 +1,303 @@
...
+  "#AuthForm_External_Instruction": "Cliquez sur Autoriser dans le navigateur",
+  "#AuthForm_External_Or_Internal": "ou saisissez les identifiants",
+  "#AuthForm_External_Failed": "Échec de la connexion, veuillez réessayer",
+  "&AuthForm_Registration_Activation_1": "Nous vous avons envoyé un e-mail d'activation,\nqui devrait arriver bientôt.",
+  "&AuthForm_Registration_Activation_2": "Une fois l'activation de votre compte terminée,\ncliquez sur le bouton de connexion.",
+  "#AuthForm_Registration_Activation_3": "Assurez-vous de vérifier votre dossier de spam si vous ne le voyez pas.",
+  "#AuthForm_Registration_ResendActivation": "Renvoyer l'e-mail d'activation",
+  "#AuthForm_Registration_ActivationSent": "E-mail d'activation envoyé",
+  "#AuthForm_Registration_ActivationError": "Échec de la réexpédition de l'e-mail d'activation, veuillez réessayer ultérieurement",
+  "#AuthForm_FieldLabel_Email": "E-mail",
```
[the remark this answers is not in the document]
Comment (Mathu-lmn) [reply ext/cfx-ui/src/assets/languages/locale-fr.json:15]: Well in france, nobody says courriel lol, we all say email :/

File: ext/cfx-ui/src/assets/languages/locale-fr.json (hunk 3 above)
Comment (Mathu-lmn) [reply ext/cfx-ui/src/assets/languages/locale-fr.json:138]: English version is missing it too

File: ext/cfx-ui/src/assets/languages/locale-fr.json (hunk 2 above)
Comment (Mathu-lmn) [reply ext/cfx-ui/src/assets/languages/locale-fr.json:46]: It's in the english version too

Comment (Mathu-lmn) [reply ext/cfx-ui/src/assets/languages/locale-fr.json:46]: image

File: ext/cfx-ui/src/assets/languages/locale-fr.json (hunk 1 above)
Comment (Mathu-lmn) [reply ext/cfx-ui/src/assets/languages/locale-fr.json:37]: You're correct

File: ext/cfx-ui/src/assets/languages/locale-fr.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (Mathu-lmn) [reply ext/cfx-ui/src/assets/languages/locale-fr.json:15]: I believe in canada, courriel is mostly used but it's not the case in france

**Comment by Morgan-Carterr** [comment]
Hello, any news on the state of this PR ?

**Comment by PsychoShock** [comment]
Hello,

I'll review it again to night and notify the right person after.

Thanks

File: ext/cfx-ui/src/assets/languages/locale-fr.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (PsychoShock) [reply ext/cfx-ui/src/assets/languages/locale-fr.json:15]: `E-mail` is probably a standart french use, but it's not a french word. That should still be changed in `Courriel` in my opinion.

File: ext/cfx-ui/src/assets/languages/locale-fr.json [hunk 5]
```diff
@@ -1,113 +1,303 @@
...
+  "#Server_FeaturedServer_Title": "Serveur en vedette",
+  "#ServerList_HideFull": "Masquer les serveurs complets",
+  "#ServerList_HideEmpty": "Masquer les serveurs vides",
+  "#ServerList_MaxPing": "Ping maximum",
+  "#ServerList_FeaturedServers": "Serveurs en vedette",
+  "#Servers_Connecting": "Connexion",
+  "#Servers_ConnectingTo": "Connexion à {{ serverName }}...",
+  "#Servers_ConnectingToServer": "Connexion au serveur...",
+  "#Servers_ConnectFailed": "Échec de la connexion",
+  "#Servers_Error": "Ошибка (Error)",
```
Comment (PsychoShock) [inline ext/cfx-ui/src/assets/languages/locale-fr.json:79]: Forgot that part.

**Review by PsychoShock — COMMENTED** [review]
Hello,

The only thing I see is missing here. I have notify the right person for the PR btw, great job.

Thanks

File: ext/cfx-ui/src/assets/languages/locale-fr.json (hunk 5 above)
Comment (Mathu-lmn) [reply ext/cfx-ui/src/assets/languages/locale-fr.json]: Thanks !

**Comment by Mathu-lmn** [comment]
Cleaned commits history to match CI/CD requirements

File: ext/cfx-ui/src/assets/languages/locale-fr.json [hunk 6]
```diff
@@ -1,113 +1,303 @@
...
+  "#Server_FeaturedServer_Title": "Serveur en vedette",
+  "#ServerList_HideFull": "Masquer les serveurs complets",
+  "#ServerList_HideEmpty": "Masquer les serveurs vides",
+  "#ServerList_MaxPing": "Ping maximum",
+  "#ServerList_FeaturedServers": "Serveurs en vedette",
+  "#Servers_Connecting": "Connexion",
+  "#Servers_ConnectingTo": "Connexion à {{ serverName }}...",
+  "#Servers_ConnectingToServer": "Connexion au serveur...",
+  "#Servers_ConnectFailed": "Échec de la connexion",
+  "#Servers_Error": "Ошибка (Erreur)",
```
[the remark this answers is not in the document]
Comment (Mathu-lmn) [reply ext/cfx-ui/src/assets/languages/locale-fr.json:79]: https://github.com/citizenfx/fivem/pull/98
For the meme !

**Comment by gottfriedleibniz** [comment]
Our localization policies are being updated and that includes less reliance on community contributions. So at this time we will not be accepting any locale changes.

I will be closing this PR for now to avoid having others spend anymore time on this.

**Comment by Mathu-lmn** [comment]
Is there a way I can contribute ?

**Comment by Mathu-lmn** [comment]
Hey, any news about the new policy ?

**Pull request closed without merging**